settings.gradle

```diff
@@ -12,11 +12,11 @@
  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
  * License for the specific language governing permissions and limitations under
  * the License.
- *
  */
 
 rootProject.name = 'azkaban'
 
+include 'az-exec-util'
```

Comment:
Will have more code check in this gradle sub project?

Reply:
Maybe, there is currently no plan to do so.